.github/ISSUE_TEMPLATE/수정-작업.md

```
@@ -0,0 +1,15 @@
+---
+name: 수정 작업
+about: 기존 기능 수정 및 개선
+title: "[MOD]"
+labels: modification
+assignees: ''
+
+---
+
+# 📍 화면/영역
+# 🔧 기능
+#  📋 수정 내용
```

markdownlint-cli2 (0.18.1)

Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)